docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx

```diff
@@ -67,44 +67,21 @@ mode.
     cellular connections a cellular indicator will appear next to the 
     wifi icon with the currently active connection being highlighted in green.
 
-10. **Views List:** A dropdown list of available views, detailed later
+10. **Stack Light:** If a Stack Light is configured for the UGV it's current status 
+    will show here.  
+
+11. **Views List:** A dropdown list of available views, detailed later
     in this section. Some of the available views are Map, Camera and 3D
     views, etc.
 
-11. **Record Audio:** If the UGV is equipped with a microphone the user 
-    can start/stop recording manually by clicking this icon.
-
-12. **Mission List:** View the list of Missions that Operator(s) have
-    created.
-
-13. **Edit Mission Toggle:** This toggle allows the Operator to start
-    creating Waypoints for the current Mission. The Operator will also be able to
-    drag and drop Waypoints in this mode. Once this button is enabled,
-    the Waypoint Mode will be available for selection. When a Mission is 
-    started this toggle will be disabled (ie. the Operator will not be able 
-    to modify/add Waypoints).
+12. **Lights:** Any light controls that are setup on the UGV can be controlled through 

```

Comment:
I would make this: Lights Control: If the UGV is equipped with software controllable lights...